### PR TITLE
Improve coverage in scanner/package

### PR DESCRIPTION
**Pull request by Serabe**
Depends on src-d/go-parse-utils#1

This improves coverage getting it up to 90%. There are things that can be improved still, but other topics are more relevant for the v1.

### CODE REVIEW

File: scanner/package_test.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	assert.Equal(t, "map[string]int", typ.UnqualifiedName(), "Map.UnqualifiedName returns a map signature")
+}
+
+// Asserts a type respond as expected to IsRepeated and SetRepeated.
+func assertRepeatable(t *testing.T, typ Type, name string) {
+	assert.False(t, typ.IsRepeated(), fmt.Sprintf("%s is not repeated by default", name))
+	typ.SetRepeated(true)
+	assert.True(t, typ.IsRepeated(), fmt.Sprintf("%s can be set as repeated", name))
+	typ.SetRepeated(false)
+	assert.False(t, typ.IsRepeated(), fmt.Sprintf("%s can be set as not repeated", name))
```
Comment (erizocosmico) [inline scanner/package_test.go]: no need for `fmt.Sprintf`, assert already does it:

```go
assert.False(t, foo, "%s is foo", foo)
```

File: scanner/package_test.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	name := "Map"
+
+	assertRepeatable(t, typ, name)
+	assertNullable(t, typ, name)
+	assert.Equal(t, "map[string]int", typ.String(), "Map.String returns a map signature")
+	assert.Equal(t, "map[string]int", typ.TypeString(), "Map.TypeString returns a map signature")
+	assert.Equal(t, "map[string]int", typ.UnqualifiedName(), "Map.UnqualifiedName returns a map signature")
+}
+
+// Asserts a type respond as expected to IsRepeated and SetRepeated.
```
Comment (erizocosmico) [inline scanner/package_test.go]: comments of functions start with the name of the function

File: rpc/rpc.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/src-d/proteus/protobuf"
 	"github.com/src-d/proteus/report"
-	"github.com/src-d/proteus/source"
+	"gopkg.in/src-d/go-parse-utils.v1"
```
Comment (mcuadros) [inline rpc/rpc.go]: external packages should have one space between the private ones

File: scanner/scanner.go [hunk 4]
```diff
@@ -11,7 +11,7 @@ import (
 	"sync"
 
 	"github.com/src-d/proteus/report"
-	"github.com/src-d/proteus/source"
+	"gopkg.in/src-d/go-parse-utils.v1"
```
Comment (mcuadros) [inline scanner/scanner.go]: same

**Comment by Serabe** [comment]
Rebased, waiting for CI to merge.

**Pull request merged**